runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1557,13 +1557,10 @@ class INTERPRETER_CLASS
 					} else {
 						((UDATA*)(((J9SFStackFrame*)_sp) + 1))[-1] |= J9SF_A0_INVISIBLE_TAG;
 					}
-#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
here as well

Reply:
and other places in this file too. Essentially we want JDK15 and earlier to remain unaffected

Reply:
Fixed